.github/container/test-jax.sh

```diff
@@ -126,24 +126,25 @@ case "${BATTERY}" in
     large)
         JOBS_PER_GPU=1
         JOBS=$((NGPUS * JOBS_PER_GPU))
-        EXTRA_FLAGS="--jobs=${JOBS} --test_env=JAX_TESTS_PER_ACCELERATOR=${JOBS_PER_GPU} --test_env=JAX_EXCLUDE_TEST_TARGETS=PmapTest.testSizeOverflow"
+        EXTRA_FLAGS="--local_test_jobs=${JOBS} --test_env=JAX_TESTS_PER_ACCELERATOR=${JOBS_PER_GPU} --test_env=JAX_EXCLUDE_TEST_TARGETS=PmapTest.testSizeOverflow"
```

Comment:
Is `local_test_jobs` making a difference in our case? My understanding is that it will make a difference if 1) we run the build and test in the same Bazel session or 2) there is a remote build involved.

Reply:
The only difference I noticed is `local_test_jobs` work with local jobs (I assume local to the current host), and `jobs` works with jobs (I assume, all jobs including jobs on remote machines). So in out case it is the same, but guys from Google advises to use `local_test_jobs`.